src/snyk/base/services/authenticationService.ts

```diff
@@ -86,12 +86,13 @@ export class AuthenticationService implements IAuthenticationService {
     return `${token.slice(0, 4)}****${token.slice(-4)}`;
   }
 
-  async updateToken(token: string): Promise<void> {
+  async updateTokenAndEndpoint(token: string, apiUrl: string): Promise<void> {
     if (!token) {
       await this.initiateLogout();
     } else {
       if (!this.validateToken(token)) return Promise.reject(new Error('The entered token has an invalid format.'));
 
```

Comment:
check if apiUrl is not empty before updating

Reply:
done